src/lib/y2storage/volume_specification_builder.rb

```diff
@@ -172,5 +174,17 @@ def fallback_for_swap
         v.max_size = DiskSize.GiB(2)
       end
     end
+
+    # Volume specification fallback for root (/)
+    #
+    # @return [VolumeSpecification]
+    def fallback_for_root
```

Comment:
As commented offline, there should not be a fallback for root volume.

Reply:
Ok, I've reverted this change.